### PR TITLE
support Scala 2.13.0-M3 (and bump to sbt 1.1.1)

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

**Comment by mdedetrich** [comment]
Seems like its just scalameter which is not published, I do however want to change to sbt-jmh at one point

**Comment by mdedetrich** [comment]
Actually nvm about the sbt-jmh part, I just noticed that scalameter is better suited for benchmarking for this project

**Comment by fyodorovski** [comment]
I sent a PR with a 2.13.0-M3 target.

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +30,7 @@ before_cache:
   - find $HOME/.ivy2/cache -name "ivydata-*.properties" -type f -delete
 cache:
   directories:
-    - $HOME/.sbt/0.13/dependency
+    - $HOME/.sbt/1.0/dependency
```
Comment (xuwei-k) [inline .travis.yml:33]: sbt 1.x does not use this directory

Comment (mdedetrich) [reply .travis.yml:33]: Thanks for the pickup, I will fix it tonight

**Comment by SethTisue** [comment]
@mdedetrich ping?

**Comment by mdedetrich** [comment]
@SethTisue Do you want to merge this or what is missing?

**Comment by SethTisue** [comment]
oh, I still needed to update the ScalaMeter dependency. done

let's see if Travis-CI likes it now, if so, this will then be mergeable afaik. (I guess you'll address Kenji's comment separately.)

**Comment by SethTisue** [comment]
hmm

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: com.storm-enroute#scalameter_2.13.0-M3;0.10-SNAPSHOT: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

**Comment by SethTisue** [comment]
ok, green now. had to add the sonatype snapshots resolver. (I hadn't needed it locally since I use a local Artifactory that has it.)

**Pull request merged**